Tests/MongoSwiftSyncTests/MongoCollectionTests.swift

```diff
@@ -272,6 +272,28 @@ final class MongoCollectionTests: MongoSwiftTestCase {
         expect(deleteManyResult).to(beNil())
     }
 
+    func testRenamed() throws {
```

Comment:
I think we should have three tests to ensure adequate coverage: one that uses `dropTarget: true`, one without it set, and one with an invalid collection name.